scripts/lochness_create_template.py

```diff
@@ -83,18 +83,18 @@ def create_lochness_template(args):
     create_keyring_template(keyring_loc, args)
 
     # write commands for the user to run after editing config and keyring
-    write_commands_needed(args.outdir, config_loc,
+    write_commands_needed(args, config_loc,
```

Comment:
If we would accept `args` totally, we should omit `args.source` from the end.

Reply:
I can do this brevity.

Reply:
thanks!

Reply:
Fixed via https://github.com/AMP-SCZ/lochness/commit/05e5ab0c267e286683fbe9ad72892a3014d9af69